src/core/sys/posix/arpa/inet.d

```diff
@@ -199,23 +199,4 @@ NOTE: The following must must be defined in core.sys.posix.arpa.inet to break
 INET6_ADDRSTRLEN // from core.sys.posix.netinet.in_
 */
 
-version( linux )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( OSX )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( FreeBSD )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Solaris )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Android )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
+enum INET6_ADDRSTRLEN = 46;
```

Comment:
When we port to a new platform, is it possible that `INET6_ADDRSTRLEN` is something else than 46? A quick google search shows that it might be 48: http://lxr.free-electrons.com/source/include/linux/inet.h#L52. On other platforms it's indeed 46: http://www.castaglia.org/proftpd/doc/devel-guide/src/utils/utils.h.html. 

So the first question is whether 46 is the right number on all platforms .The second is whether just defining it 46 for all platforms improves the state of affairs. With the previous code we'd have an undefined identifier error. With the new code we'd just have a constant that may be wrong.

I stopped here. For all cases there must be a clear case the code will fail (either at compile time or link time) if the platform has not been ported properly yet. Silent defaults are dangerous.